### PR TITLE
add Selectivity.js v2.0.0

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/arendjr/selectivity
**Watch 36
  Star 829
 Fork 64**
@Piicksarn could u help me check this PR for #6173 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/Selectivity.js/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "2.0.0",
+  "filename": "selectivity-full.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/arendjr/selectivity.git"
+  },
+  "license": "MIT",
+  "description": "Modular and light-weight selection library for jQuery and Zepto.js.",
+  "keywords": [
+    "Selectivity"
```
Comment (PeterDaveHello) [inline ajax/libs/Selectivity.js/package.json]: No need to use its name as keywords, try to grab it from its description.

**Comment by maruilian11** [comment]
@Piicksarn i have added some keywords, could you help me check the PR?

**Comment by Piicksarn** [comment]
@PeterDaveHello, I think the pr is fine!
Would you mind to have a look at the pr?
Thank you!

**Comment by maruilian11** [comment]
@Piicksarn sorry, could you check the PR again?

**Comment by Piicksarn** [comment]
@PeterDaveHello, I think the pr is fine!
Would you mind to have a look at the pr?
Thank you!

File: ajax/libs/Selectivity.js/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "type": "git",
+    "url": "git://github.com/arendjr/selectivity.git"
+  },
+  "license": "MIT",
+  "description": "Modular and light-weight selection library for jQuery and Zepto.js.",
+  "keywords": [
+    "custom",
+    "template",
+    "loading indicators"
+  ],
```
Comment (PeterDaveHello) [inline ajax/libs/Selectivity.js/package.json]: I have no idea with these keywords.

**Pull request merged**